src/unix/linux/cpu.rs

```diff
@@ -437,6 +437,7 @@ pub(crate) fn get_cpu_frequency(cpu_core_index: usize) -> u64 {
     }
     let find_cpu_mhz = s.split('\n').find(|line| {
         line.starts_with("cpu MHz\t")
+            || line.starts_with("CPU MHz\t")
```

Comment:
There is a `is_key` function in this file. Move its declaration outside of the function and use it here instead.

Reply:
Done